src/main/java/edu/ucdavis/dss/ipa/api/components/assignment/views/AssignmentExcelView.java

```diff
@@ -64,16 +64,14 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
                 SectionGroup sectionGroup = this.getSectionGroupByCourseAndTermCode(course, state.getTermCode());
 
                 if (sectionGroup != null) {
-                    excelHeader.createCell(col).setCellValue(
-                            StringUtils.join(
-                                    sectionGroup.getTeachingAssignments()
-                                            .stream()
-                                            .filter(TeachingAssignment::isApproved)
-                                            .map(ta -> ta.getInstructor().getLastName() + " " + ta.getInstructor().getFirstName().charAt(0))
-                                            .collect(Collectors.toList())
-                                    , ", "
-                            )
-                    );
+                    String instructorNames = "";
+                    for (TeachingAssignment teachingAssignment : sectionGroup.getTeachingAssignments()) {
+                        String name = teachingAssignment.getInstructor() != null ? teachingAssignment.getInstructor().getLastName() + " " + teachingAssignment.getInstructor().getFirstName().charAt(0) : teachingAssignment.getInstructorType().getDescription();
+
+                        instructorNames += instructorNames.length() > 0 ? ", " + name : name;
```

Comment:
Use Java 8's String.join() with an array instead of this method. It's not necessary to do string concatenation in this older style anymore.